src/map/utils/synthutils.cpp

```diff
@@ -502,64 +424,71 @@ int32 doSynthSkillUp(CCharEntity* PChar)
         int32  baseDiff   = PChar->CraftContainer->getQuantity(skillID-40) - charSkill/10; //the 5 lvl difference rule for breaks does NOT consider the effects of image support/gear
 
         if ((baseDiff <= 0) || ((baseDiff > 5) && (PChar->CraftContainer->getQuantity(0) == SYNTHESIS_FAIL))) // synthesis result is stored in the zero cell quantity
-        {
+        {                                                                                                     // if you break a recipe higher than 5 levels, no skill ups
             continue;
         }
 
-        if (charSkill < maxSkill)
+        if (charSkill < maxSkill) // Check if a character can skill up
         {
-            double skillUpChance = ((double)baseDiff*(map_config.craft_chance_multiplier - (log(1.2 + charSkill/100))))/10;
+            double skillUpChance = (double)baseDiff * map_config.craft_chance_multiplier * (3 - (log(1.2 + charSkill/100))) / 10;
```

Comment:
Make sure I don't forget to compile my retail data~!

This is fine for now as it appears to be fixing the formula so that a setting of 1.0 is adjusted back to what the default used to be. But I'm not entirely convinced the original formula's _approach_ matches retail.

Reply:
On a future PR I was planning on updating this(skill up rate) and the skill up range with retails current system. As of now, this is.... "good enough"

Reply:
Agreed - the fewer things fixed per PR the better! 😄